janusgraph-test/src/main/java/org/janusgraph/TestCategory.java

```diff
@@ -12,13 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package org.janusgraph.testcategory;
+package org.janusgraph;
 
-/**
- * This is a JUnit category for tests that need to run against Cassandra
- * configured for SSL-based client authentication.
- *
- * If you rename or move this class, then you must also update mentions of it in
- * the Cassandra module's pom.xml.
- */
-public interface CassandraSSLTests { }
+public final class TestCategory {
+    public static final String BRITTLE_TESTS = "BRITTLE_TESTS";
```

Comment:
Actually I was talking about the variable names and not values. I.e. values like `BritleTests` are OK. It is more like a preference of choice. As fore me the perfect constant would be:
`public static final String BRITTLE_TESTS = "BritleTests";`
but current constants are OK too. You can use either proposed constants or leave these constants as you wish.

Reply:
As I understand (possibly wrong) you use the value of the variable in `pom` right? Or you use the name or the variable? I didn't get why the name of the variable and the value in the variable should be equal.

Reply:
Content of the variable is the tag and not the variable.